src/cli/index.js

```diff
@@ -26,9 +26,9 @@ export default (config, http) => {
     logger.info(`Sent report to ${email}`);
   };
 
-  const calcFlexTime = async (email) => {
-    logger.info(`Calculating flextime for ${email}`);
-    const data = await app.calcFlextime(email);
+  const calcFlexTime = async (email, startDate, endDate, calcAll) => {
+    logger.info(`Calculating flextime for ${email} ${startDate} ${endDate} ${calcAll ? 'all hours included' : ''}`);
```

Comment:
calcAll? What is the purpose, !calcAll counts also hours marked on public holidays? Should you define a default value that is aligned with previous functionality?

Originally ignoring of public holidays was implemented on purpose, since a lot of people recorded hours afterwards and entered working hours also to public holidays which mixed up the billable hours calculation. So I probably wouldn't allow people marking hours on public holidays.